public/app/percona/shared/core/reducers/nodes/nodes.utils.ts

```diff
@@ -51,6 +51,20 @@ export const nodeFromDbMapper = (nodeFromDb: NodeDB[]) => {
       })),
       properties: properties,
       agentsStatus: getAgentsMonitoringStatus(agents ?? []),
-    } as Node;
+    };
   });
 };
+
+export const nodesOptionsMapper = (nodeFromDb: NodeDB[]): NodesOption[] => {
+  return  nodeFromDb.map((node) => {
```

Comment:
Just a small stylistic improvement, you can omit the `return` statement when you directly return the result from `nodeFromDb.map`.

```
export const nodesOptionsMapper = (nodeFromDb: NodeDB[]): NodesOption[] =>
   nodeFromDb.map(...);
```

Also it looks like there are two spaces between the `return` and `nodeFromDb.map`, please make sure to run a linter/formatter.